src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars">
```

Comment:
If you dropped the container for all the groups and did as I suggested in the screenshot you could simplify you css like so:
```
.stars {
  display: flex;
  justify-content: space-between;
  width: 100px;
}

.stars__star {
  background-image: url(./images/star.svg);
  background-position: center;
  background-repeat: no-repeat;
  height: 16px;
  width: 16px;
}

.stars--1 > .stars__star:nth-child(-n+1),
.stars--2 > .stars__star:nth-child(-n+2),
.stars--3 > .stars__star:nth-child(-n+3),
.stars--4 > .stars__star:nth-child(-n+4),
.stars--5 > .stars__star:nth-child(-n+5) {
  background-image: url(./images/star-active.svg);
}
```

Reply:
I applied simplier css code, which pass tests.